src/webgpu/web_platform/reftests/canvas_size_different_with_back_buffer_size.html

```diff
@@ -0,0 +1,20 @@
+<html class="reftest-wait">
+  <title>WebGPU canvas_back_buffer_different_size</title>
+  <meta charset="utf-8" />
+  <link rel="help" href="https://gpuweb.github.io/gpuweb/" />
+  <meta
+    name=fuzzy
+    content="128;48"
+  />
```

Comment:
Per https://web-platform-tests.org/writing-tests/reftests.html#fuzzy-matching
this are very lenient settings. It may be good enough, but do we really need it to be so lenient? Does it really need to be fuzzy at all?

Reply:
You're right, we don't need be fuzzy at all after applying CSS!
(My state: Idiot on CSS -- -Level up! ---> rookie on CSS :P )